module/VuFind/src/VuFind/Db/Row/UserList.php

```diff
@@ -93,6 +103,32 @@ public function getTags()
         return $user->getTags(null, $this->id);
     }
 
+    /**
+     * Get an array of resource tags associated with this list
+     * (deprecated, use getResourceTags).
```

Comment:
You can use a `@deprecated` annotation in the comment, which might improve the PHPDoc (and make it easier for me to find and remove this in the future).